src/main/java/interface_adapter/translateText/TranslateTextPresenter.java

```diff
@@ -19,8 +19,8 @@ public TranslateTextPresenter(TranslateTextViewModel translateTextViewModel) {
      * @param translatedText the output data
      */
     @Override
-    public void prepareSuccessView(String translatedText, String inputLanguage) {
-        translateTextViewModel.getState().setText(translatedText);
+    public void prepareSuccessView(String outputText, String inputLanguage) {
+        translateTextViewModel.getState().setText(outputText);
```

Comment:
you should call the .setOutputText of the State here, besides that everything looks good